Marlin/src/lcd/menu/menu.cpp

```diff
@@ -269,6 +269,13 @@ void scroll_screen(const uint8_t limit, const bool is_menu) {
   }
   else
     encoderTopLine = encoderLine;
+  #ifdef MENU_SCROLL_FEEDBACK
+    static int8_t last_encoderLine = encoderLine;
+    if (encoderLine != last_encoderLine) {
+      BUZZ(2, 15000);
```

Comment:
It'd be good to reuse the existing LCD feedback defines here (or create new ones for this new type of feedback):

https://github.com/MarlinFirmware/Marlin/blob/4a4c1db606ece3a235244c90fc397946d0c863a6/Marlin/Configuration.h#L2724-L2725

Reply:
Oh yes I actually forgot to do that. Will fix